allocator/allocator_test.go

```diff
@@ -382,8 +382,8 @@ type releasePeer struct {
 }
 
 type pendingResult struct {
-	p      peer.ID
-	amount uint64
+	p      peer.ID //nolint:structcheck
+	amount uint64  //nolint:structcheck
```

Comment:
what warning is this preventing? as far as I know our unified CI only enforces vet and staticcheck, so structcheck should not be a part of any of this.

Reply:
it gets the CURRENT CI to pass :P

Reply:
:facepalm: need to rush to remove golangci-lint, then :P

Reply:
I'll remove the nolint comment in the PR where we switch over.